### DISCUSSION TITLE
Disable Highlights for specific channels

### DISCUSSION THREAD

**Discussion by Fabo0814** [Chatterino/chatterino2#4282] [Ideas] [answered]
A setting which disables highlights from specific channels or specific users.

**Comment by pajlada** [comment, marked as answer]
You can disable all highlights from specific users by opening their usercard (this can be done by clicking their name or by typing `/usercard theirtwitchlogin`), then checking the "Ignore highlights" setting.

The original question I'd argue is a duplicate of [this feature request](https://github.com/Chatterino/chatterino2/issues/2365), just the inverse
